### DISCUSSION TITLE
✅ [Submitted] OSPO DevRoom - FOSDEM 2023

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#191] [Community Proposals]
Would people be interested to propose an OSPO DevRoom for this year? The deadline is October 18th, and if there is interest, I can make the submission. Also, if anyone would like to help out on this as a DevRoom Manager, please DM me on TODO slack!

Here is an [initial draft for submission](https://submission.fosdem.org/submission/devroom). Please use this discussion thread to propose changes, additions, or any other ideas in mind ☺️:

## Title of devroom

Open Source Offices & Programs

## Elaborate description of proposed devroom 

An [Open Source Program Office (OSPO)](https://github.com/todogroup/ospodefinition.org#ospo-definition) is implemented in [many organizations](https://landscape.todogroup.org/) to act as an open source catalyzer. They are a designated place where open source is supported, nurtured, shared, explained, and grown inside an organization. With such an office in place, organizations can establish and execute their open source strategies and policies with clear terms and responsibilities, giving their leaders, developers, lawyers, marketers, and other staff the processes and tools they need to make open source a success within their operations and build healthy relationships with the open source ecosystem. 

Topics can be structured  into these [key areas](https://ospomindmap.todogroup.org/):

📚 Develop and Execute Open Source Strategy
🧭 Eliminate Friction from Using and Contributing to Open Source
🖥️ Manage Open Source IT Infrastructure
🧑‍💼 Give Advice on Open Source
🫶 Grow and Retain Open Source Talent Inside the Organization
🤝 Implement InnerSource Practices
⏱️ Track Performance Metrics
🤝 Collaborate with Open Source Organizations
📈 Prioritize and Drive Open Source Upstream Development
💪 Establish and Improve Open Source Processes
📝 Establish and Improve Open Source Policies
🔍 Oversee Open Source Compliance
📒 Support Corporate Development Activities

## Why does it fit FOSDEM?

We believe an Open Source Offices & Programs Devroom will fit FOSDEM because their massive adoption can be key for open source sustainability and promote the widespread of open source software. For instance, we've seen OSPOs releasing projects as open source, guiding and training their developers to contribute back to existing open source projects, or creating fund programs to help open source maintainers. 

**********

## FYI

* Devroom managers must enter a complete schedule into the conference system by 15th December.

* Developer room proposals should be submitted through the form at https://fosdem.org/devroom which contains further information.

* Key dates:

    18 October
        deadline for developer room proposals
    31 October
        accepted developer rooms announced
    5 November (or earlier)
        developer rooms issue Calls for Participation
    15 December (or earlier)
        developer rooms publish complete schedules

**Comment by caniszczyk** [comment]
An OSPO devrroom at FOSDEM is overdue imho

**Comment by tsteenbe** [comment]
@anajsana  Happy to help co-organize an OSPO DevRoom with you ...

**Comment by Claudenw** [comment]
Has anyone submitted a proposal?  Deadline is in 5 days.

**Comment by anajsana** [comment]
I can take care of it and make the submission tomorrow. We have 3 people interested to help and become devrom managers 🙂

If you'd like to propose changes to the devroom description, please share it here 😉

**Reply by Claudenw** [reply]
I tried to reach out to you on Slack.  Where can I see the submission proposal?

**Reply by jlprat** [reply]
Hi Claude, I think it's the description of this discussion, see the text below "Please use this discussion thread to propose changes, additions, or any other ideas in mind relaxed"

**Comment by anajsana** [comment]
DevRoom form submitted and created a Slack DM group with people who show interest in helping with OSPO DevRoom management:

* @tsteenbe 
* @Claudenw 
* Divya Mohan
